src/program.js

```diff
@@ -124,10 +126,19 @@ export class Program {
 }
 
 
-export function defaultVersionGetter(absolutePackageDir: string): string {
-  let packageData: any = readFileSync(
-    path.join(absolutePackageDir, 'package.json'));
-  return JSON.parse(packageData).version;
+export function defaultVersionGetter(
+  absolutePackageDir: string,
+  localProcess: Object = process
+): string {
+  if (localProcess.env.NODE_ENV === 'production') {
+    log.debug('Getting the version from package.json');
```

Comment:
oops, the code needs this fix to make the log statements show up in `--verbose` mode:
````diff
diff --git a/src/program.js b/src/program.js
index eb9e8ae..5bf90ae 100644
--- a/src/program.js
+++ b/src/program.js
@@ -92,8 +92,8 @@ export class Program {
     let runCommand = this.commands[cmd];
 
     if (argv.verbose) {
-      log.info('Version:', getVersion(absolutePackageDir));
       logStream.makeVerbose();
+      log.info('Version:', getVersion(absolutePackageDir));
     }
 
     try {
````